vita3k/gui/src/user_management.cpp

```diff
@@ -232,91 +238,119 @@ static auto get_users_index(GuiState &gui, const std::string &user_name) {
+        ImGui::GetBackgroundDrawList()->AddImage(gui.apps_background["NPXS10013"], WINDOW_POS, WINDOW_POS_MAX);
     else
-        ImGui::GetBackgroundDrawList()->AddRectFilled(ImVec2(0.f, INFORMATION_BAR_HEIGHT), display_size, IM_COL32(10.f, 50.f, 140.f, 255.f), 0.f, ImDrawFlags_RoundCornersAll);
+        ImGui::GetBackgroundDrawList()->AddRectFilled(WINDOW_POS, WINDOW_POS_MAX, IM_COL32(10.f, 50.f, 140.f, 255.f), 0.f, ImDrawFlags_RoundCornersAll);
 
     const auto user_path{ fs::path(emuenv.pref_path) / "ux0/user" };
-    const auto SMALL_AVATAR_SIZE = (ImVec2(avatar_small_size * SCALE.x, avatar_small_size * SCALE.y));
+
+    const auto SMALL_AVATAR_SIZE = get_avatar_size(SMALL, SCALE);
+    const auto MED_AVATAR_SIZE = get_avatar_size(MEDIUM, SCALE);
+    const auto LARGE_AVATAR_SIZE = get_avatar_size(LARGE, SCALE);
+    const auto USER_NAME_BG_SIZE = ImVec2(MED_AVATAR_SIZE.x, 60.f * SCALE.y);
+
     ImGui::SetWindowFontScale(1.4f * RES_SCALE.x);
     const auto calc_title = ImGui::CalcTextSize(title.c_str()).y / 2.f;
     ImGui::SetCursorPos(ImVec2(54.f * SCALE.x, (32.f * SCALE.y) - calc_title));
     ImGui::TextColored(GUI_COLOR_TEXT, "%s", title.c_str());
 
-    const auto AVATAR_SIZE = ImVec2(avatar_size * SCALE.x, avatar_size * SCALE.y);
-    const auto SIZE_USER = ImVec2(960.f * SCALE.x, 376.f * SCALE.y);
-    const auto POS_SEPARATOR = 68.f * SCALE.y;
-    const auto SPACE_AVATAR = AVATAR_SIZE.x + (20.f * SCALE.x);
+    const auto SIZE_USER = ImVec2(WINDOW_SIZE.x, 408.f * SCALE.y);
+    const auto POS_SEPARATOR = 66.f * SCALE.y;
+    const auto SPACE_AVATAR = MED_AVATAR_SIZE.x + (20.f * SCALE.x);
 
     ImGui::SetCursorPosY(POS_SEPARATOR);
     ImGui::Separator();
 
-    if (menu.empty())
-        ImGui::SetNextWindowContentSize(ImVec2(SIZE_USER.x + ((gui.users.size() - 2.5f) * SPACE_AVATAR), 0.0f));
-    ImGui::SetNextWindowPos(ImVec2(0.f, 102.f * SCALE.y), ImGuiCond_Always);
+    if (menu == SELECT)
+        ImGui::SetNextWindowContentSize(ImVec2(SIZE_USER.x + ((gui.users.size() - 2.6f) * SPACE_AVATAR), 0.0f));
+    ImGui::SetNextWindowPos(ImVec2(WINDOW_POS.x, WINDOW_POS.y + (68.f * SCALE.y)), ImGuiCond_Always);
     ImGui::BeginChild("##user_child", SIZE_USER, false, ImGuiWindowFlags_NoBackground | ImGuiWindowFlags_HorizontalScrollbar | ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoCollapse | ImGuiWindowFlags_NoSavedSettings);
 
-    const auto AVATAR_POS = ImVec2((SIZE_USER.x / 2) - (AVATAR_SIZE.x / 2.f), (((menu == "create" || menu == "edit") ? 48.f * SCALE.y : (SIZE_USER.y / 2) - (AVATAR_SIZE.y / 2.f))));
-    const auto NEW_USER_POS = AVATAR_POS.x - (gui.users.size() ? SPACE_AVATAR : 0.f);
-    const auto DELETE_USER_POS = AVATAR_POS.x + (SPACE_AVATAR * (gui.users.size()));
+    // Clear temp user
+    const auto clear_temp_user = [&]() {
+        temp = {};
+        gui.users_avatar["temp"] = {};
+        gui.users_avatar.erase("temp");
```

Comment:
No need to clear before erase

Reply:
is obligatory, in moment it run ths, picture is keep display
need clear texcture before can erase it